apps/site/lib/site_web/controllers/schedule/line/helpers.ex

```diff
@@ -18,12 +18,18 @@ defmodule SiteWeb.ScheduleController.Line.Helpers do
 
   @spec get_route(String.t()) :: {:ok, Route.t()} | :not_found
   def get_route(route_id) do
```

Comment:
I feel like pattern matching might be a better fit here - e.g. adding the function `def get_route(""), do: :not_found`